toolbox_runner/tool.py

```diff
@@ -306,9 +306,15 @@ def _build_parameter_file(self, path: str, **kwargs) -> str:
             # Copy any file source
             elif isinstance(value, str):
                 if self.parameters[key]['type'] == 'file':
-                    fname = f"{key}{os.path.splitext(value)[1]}"
-                    shutil.copy(value, os.path.join(path, fname))
-                    value = f"/in/{fname}"
+                    # check if file is actually a folder (no file extension)
+                    if not os.path.splitext(value)[1]:
```

Comment:
you can use os to check if a folder is a folder. Otherwise this does not handle files without extension or files without filename correctly...